crates/tabby/src/main.rs

```diff
@@ -124,22 +135,28 @@ async fn main() {
 
     let config = Config::load().unwrap_or_default();
 
-    match &cli.command {
-        Commands::Serve(args) => serve::main(&config, args).await,
-        Commands::Download(args) => download::main(args).await,
-        Commands::Scheduler(args) => tabby_scheduler::scheduler(args.now, &config)
+    match cli.command {
+        Commands::Serve(ref args) => serve::main(&config, args).await,
+        Commands::Download(ref args) => download::main(args).await,
+        Commands::Scheduler(args) => tabby_scheduler::scheduler(args.now, config.repositories)
             .await
             .unwrap_or_else(|err| fatal!("Scheduler failed due to '{}'", err)),
         #[cfg(feature = "ee")]
-        Commands::JobSync => tabby_scheduler::job_sync(&config),
+        Commands::JobSync(args) => {
+            let repositories = get_repositories(args.url, args.token, &config.repositories).await;
+            tabby_scheduler::job_sync(&repositories)
```

Comment:
i still prefer this block being moved to job.rs, with a function name like `job_sync`